extensions/amp-form/0.1/amp-form.js

```diff
@@ -192,7 +197,10 @@ function installSubmissionHandlers(win) {
 function installAmpForm(win) {
   return getService(win, 'amp-form', () => {
     if (isExperimentOn(win, TAG)) {
-      installStyles(win.document, CSS, () => installSubmissionHandlers(win));
+      installStyles(win.document, CSS, () => {
+        installSubmissionHandlers(win);
+        actionServiceFor(win).addEvent('submit');
```

Comment:
Can we introduce both `submitSuccess` and `submitError` events ( @cramforce mentioned this in a comment above as well)? 

Not sure how one would use `ActionService.trigger` but will look into it. Does that mean we don't need to `addEvent('submit')` or does that still need to happen?